### PR TITLE
Add OpsWorks section.

### LINKED ISSUE #123: Start section: OpsWorks
Basics around when/how it's useful and a few tips, to get this section started.

### PR DESCRIPTION
**Pull request by ClydeMachine**
Create basic OpsWorks section with some general details and gotchas. It's a cleaned up version of previous PR [#348](https://github.com/open-guides/og-aws/pull/348). Fixes #123.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1728,6 +1729,37 @@ WAF
...
+- Major competitors to Chef include [Puppet](https://puppet.com/product/how-puppet-works) and [Ansible](https://www.ansible.com/how-ansible-works).
+
+### OpsWorks Tips
+
+- [OpsWorks relies heavily on Chef cookbooks and recipes](http://docs.aws.amazon.com/opsworks/latest/userguide/gettingstarted-cookbooks.html) for customization, so familiarity with reading their syntax will help greatly with getting up and running.
+
+### OpsWorks Gotchas and Limitations
+
+- Although OpsWorks will let you work with common Chef recipes when creating your stacks, creating custom recipes will require familiarity with Chef syntax.
+- As of December 2016, OpsWorks is available in all regions except Canada, GovCloud (US), and China (Beijing).
```
Comment (ThanosBaskous) [inline README.md]: nit (can change after merge): has this changed?

File: README.md [hunk 2]
```diff
@@ -1728,6 +1729,37 @@ WAF
...
+- As of December 2016, OpsWorks for ChefAutomate uses [Chef Server version 12.11.1](http://docs.aws.amazon.com/opsworks/latest/userguide/welcome_opscm.html) This is the current stable version of Chef.
+- [Berkshelf can be used](http://docs.aws.amazon.com/opsworks/latest/userguide/workingcookbook-chef11-10.html#workingcookbook-chef11-10-berkshelf) with Chef stacks of version 11.10 and later for managing cookbooks and their respective dependencies.
+- Running your own Chef environment may be an alternative to consider - some considerations are listed [in this Bitlancer article.](http://www.bitlancer.com/blog/2015/10/05/opsworks-vs-chef.html)
+- A key difference between running OpsWorks and rolling your own Chef environment is that the latter allows the scheduling of Chef runs, while the former has Chef runs occur according to lifecycle hooks.
+
+### OpsWorks Alternatives and Lock-in
+
+- Major competitors to Chef include [Puppet](https://puppet.com/product/how-puppet-works) and [Ansible](https://www.ansible.com/how-ansible-works).
```
Comment (QuinnyPig) [inline README.md]: Nit: What, no Saltstack?

File: README.md (hunk 1 above)
Comment (QuinnyPig) [inline README.md]: Opsworks Stacks isn't available in Canada, GovCloud, and Beijing. 
Opsworks for Chef Automate is missing in Ohio, California, Montreal, Sao Paolo, Ireland, Frankfurt, London, and all of the Asia Pacific regions.

**Comment by QuinnyPig** [comment]
Thanks. Merged-- feel free to adjust any of the above nits.

File: README.md (hunk 1 above)
Comment (ClydeMachine) [inline README.md]: Addressed this in #428. Found that the Ireland region [does support, as of December 2016](https://aws.amazon.com/blogs/aws/new-aws-opsworks-for-chef-automate/), OpsWorks for Chef Automate.

**Pull request merged**